app/helpers/organisation_helper.rb

```diff
@@ -28,7 +28,13 @@ def organisation_types(organisations)
   end
 
   def organisation_type_basic(organisation)
-    organisation.is_a?(School) ? 'school' : 'trust'
+    if organisation.is_a?(School)
+      'school'
+    elsif organisation.group_type == 'local_authority'
+      'local authority'
```

Comment:
shouldnt this be `local_authority` like in the vacancies component line 67 above?